src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -8508,6 +8508,7 @@ def test_aks_approuting_enable_with_keyvault_secrets_provider_addon(self, resour
         delete_cmd = 'aks delete --resource-group={resource_group} --name={aks_name} --yes --no-wait'
         self.cmd(delete_cmd, checks=[self.is_empty()])
 
+    @live_only()
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=84038786&view=results) to validate the change, test passed!

Reply:
Discussed offline, as the role assignment operations could not be recorded and replayed correctly, marked the case as live only. Better left a comment here